fenix/app/src/main/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckService.kt

```diff
@@ -43,6 +44,13 @@ interface ReviewQualityCheckService {
      * Returns the selected tab url.
      */
     fun selectedTabUrl(): String?
+
+    /**
+     * Fetches product recommendations related to the product user is browsing in the current tab.
+     *
+     * @return [ProductRecommendation] if request succeeds, null otherwise.
+     */
+    suspend fun productRecommendations(): ProductRecommendation?
```

Comment:
the plurality/singularity verbiage here is mismatched. The API returns a list of recommendations, but based on the default implementation, we only ever want the first one that's returned?

Reply:
Indeedio, it was returning a list earlier and then middleware was responsible for the logic of getting the first one but makes sense to do it here at the edge since we only care about the first recommendation.